zh/prerequisites.md

```diff
@@ -163,41 +165,42 @@ sysctl --system
 
 安装完 Docker 服务以后，执行以下步骤：
 
-1. 将 Docker 的数据保存到一块单独的盘上，Docker 的数据主要包括镜像和容器日志数据。通过设置 [`--data-root`](https://docs.docker.com/config/daemon/systemd/#runtime-directory-and-storage-driver) 参数来实现：
+将 Docker 的数据保存到一块单独的盘上，Docker 的数据主要包括镜像和容器日志数据。通过设置 [`--data-root`](https://docs.docker.com/config/daemon/systemd/#runtime-directory-and-storage-driver) 参数来实现：
```

Comment:
It's not recommended to remove the list number, because the user might unconsciously skip certain steps.